adapters/Unity/UnityAdapter/AdNetworkAdapterProxy/GADMUnityBannerNetworkAdapterProxy.m

```diff
@@ -43,4 +43,7 @@ - (void)bannerViewDidLeaveApplication:(UADSBannerView *)bannerView {
   [self.connector adapterWillLeaveApplication:self.adapter];
 }
 
+- (void)bannerViewDidShow:(UADSBannerView *)bannerView {
+}
```

Comment:
@vita-unity 

Can you remind me again why we are not calling GMA SDK's `reportImpression` here?

Reply:
@joshh-devrel could you please advise which method we should call here? [GADMAdNetworkConnector](https://developers.google.com/admob/ios/api/reference/Protocols/GADMAdNetworkConnector) doesn't have `reportImpression`